src/main.ts

```diff
@@ -1,39 +1,80 @@
 import * as core from '@actions/core';
 import * as fs from 'fs';
+import path from 'path';
 
 import {getAngularVersions} from './get-angular-versions';
 import {overrideAngularVersions} from './override-angular-versions';
+import {replaceLibrariesBuildBuilder} from './replace-libraries-build-builder';
+import {AngularJson} from './types/angular-json';
 import {PackageJsonVersion} from './types/package-json-version';
 
+function ensureCorrectLibraryBuildBuilder(
+  angularVersion: string,
+  angularJsonPath: string
+): void {
+  core.debug(
+    `Accessing angular.json and replacing all libraries build builder with the corresponding builder for version ${angularVersion}`
+  );
+  const angularJson: AngularJson = JSON.parse(
+    fs.readFileSync(angularJsonPath).toString()
+  );
+  const modifiedAngularJson = replaceLibrariesBuildBuilder(
+    angularVersion,
+    angularJson
+  );
+  fs.writeFileSync(
+    angularJsonPath,
+    JSON.stringify(modifiedAngularJson, null, 2)
+  );
+  core.debug(
+    'All libraries build builder successfully modified to match the angular version'
```

Comment:
suggestion: use a more precise description and don't use the term *library build builder*.
```suggestion
    'ng-packagr targets replaced to be compatible with the specified Angular version'
```